openapi/components/schemas/RiskScore/RiskScoreBoolean.yaml

```
@@ -0,0 +1,8 @@
+type: object
+nullable: true
+required:
+  - scoreValue
+properties:
+  scoreValue:
+    type: integer
+    description: This value is added to transaction risk score.
```

nitpick: suggestion to make it more concise
```suggestion
    description: Value added to the risk score of the transaction.
```